code/modules/research/rdconsole.dm

```diff
@@ -498,28 +501,37 @@ won't update every console in existence) but it's more of a hassle to do. Also,
 					message_admins("Circuit imprinter exploit attempted by [key_name(usr, usr.client)]!")
 
 				if(g2g) //Again, if input is wrong, do nothing
-					add_wait_message("Imprinting Circuit. Please Wait...", IMPRINTER_DELAY)
+					var/time_to_construct = IMPRINTER_DELAY //Flat 1.6 seconds
+					var/enough_materials = 1
+					
+					add_wait_message("Imprinting Circuit. Please Wait...", time_to_construct)

```

Comment:
It is rather moot functionally. I initially created that thinking there was supposed to be more to it, then abanded that after switching it back to just IMPRINTER_DELAY, then kept it after thinking it was easier to follow.